src/client/main.tsx

```diff
@@ -61,12 +64,33 @@ export class Main extends React.Component<{}, ClientState> {
   startGame(initialState: GameState, playerId: string) {
       this.gameState = initialState;
       this.activePlayer = playerId;
-      this.controller = new ClientController(this.gameState, this.socket);
+      this.controller = new ClientController(this.gameState, this.socket, (winner: string) => {
+        this.socket.disconnect()
```

Comment:
Can we move this stuff into a different function, its awkward to have all of this stuff stuffed into `startGame`